client/components/Item/UpNext.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <v-fade-transition>
+    <v-card class="container white--text pa-4">
+      <div class="d-flex flex-column flex-grow-1">
+        <v-card-title class="text-h6 pa-0 my-1 mx-0">
+          {{ $t('dialog.upNext.nextItemPlayingIn') }}
+          <span class="primary--text darken-2">
+            &ensp;{{ timeLeft }} {{ $t('seconds').toLowerCase() }}
```

Comment:
This isn't properly localized.

Use something like `$tc('units.time.seconds', { value: timeLeft })`

Not only does this allow you to use [pluralization](https://kazupon.github.io/vue-i18n/guide/pluralization.html) properly, it also allows translator to move the unit if they need to (Currently, you force the unit to be after the number, which may not always be the case).